kroxylicious-app/src/main/java/io/kroxylicious/app/Kroxylicious.java

```diff
@@ -127,17 +126,8 @@ public static void main(String... args) {
     static class VersionProvider implements CommandLine.IVersionProvider {
         @Override
         public String[] getVersion() throws Exception {
-            try (InputStream resource = this.getClass().getClassLoader().getResourceAsStream("META-INF/metadata.properties")) {
-                if (resource != null) {
-                    Properties properties = new Properties();
-                    properties.load(resource);
-                    String version = properties.getProperty("kroxylicious.version", UNKNOWN);
-                    String commitId = properties.getProperty("git.commit.id", UNKNOWN);
-                    String commitMessage = properties.getProperty("git.commit.message.short", UNKNOWN);
-                    return new String[]{ "kroxylicious: " + version, "commit id: " + commitId, "commit message: " + commitMessage };
-                }
-            }
-            return new String[]{ UNKNOWN };
+            var versionInfo = VersionInfo.VERSION_INFO;
+            return new String[]{ "kroxylicious: " + versionInfo.version(), "commit id: " + versionInfo.commitId(), "commit message: " + versionInfo.commitMessage() };
```

Comment:
It's not directly related to this PR, but I don't understand why we want the commit message in the `versionInfo`. I don't understand how that's useful to anyone.

Reply:
Agreed, I'm for taking commit message away.